docker/utils/utils.py

```diff
@@ -509,6 +509,13 @@ def create_host_config(
     if devices:
         host_config['Devices'] = parse_devices(devices)
 
+    if group_add:
+        if compare_version(version, '1.20') < 0:
```

Comment:
yup, I think I fixed it afterwards but it may be in one of the pending PRs. Let me find it.